doc/rst/language/spec/interoperability.rst

```diff
@@ -244,14 +232,22 @@ declaration with the following syntax.
 .. code-block:: syntax
 
    external-type-alias-declaration-statement:
-     'extern' 'type' type-alias-declaration-list ;
+     'extern' external-name[OPT] 'type' type-alias-declaration-list ;
 
 In each ``type-alias-declaration``, if the ``type-expression`` part is
 supplied, then Chapel uses the supplied type specifier internally.
 Otherwise, it treats the named type as an opaque type. The definition
 for an external type must be supplied by a C header file named on the
 ``chpl`` command line.
 
+If an optional ``external-name`` is supplied, it provides the name of
+the symbol(s) in C, where the given Chapel identifier(s) are used to
+refer to them within Chapel code.
+
+is the name
+used when interacting with external code, while it is used
+verbatim as the name of the external type alias.
```

Comment:
Did you mean to delete this paragraph? If not it needs some help.

Reply:
D'oh, yes!